src/course-outline/configure-modal/BasicTab.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Stack } from '@edx/paragon';
+import { FormattedMessage, injectIntl, useIntl } from '@edx/frontend-platform/i18n';
+import messages from './messages';
+import { DatepickerControl, DATEPICKER_TYPES } from '../../generic/datepicker-control';
+
+const BasicTab = ({ releaseDate, setReleaseDate }) => {
+  const intl = useIntl();
+  const onChange = (value) => {
+    setReleaseDate(value);
+  };
+
+  return (
+    <>
+      <h3 className="mt-3"><FormattedMessage {...messages.releaseDateAndTime} /></h3>
+      <hr />
+      <Stack direction="horizontal" gap={5}>
+        <DatepickerControl
+          type={DATEPICKER_TYPES.date}
+          value={releaseDate}
+          label={intl.formatMessage(messages.releaseDate)}
+          controlName="state-date"
+          onChange={(date) => onChange(date)}
+        />
+        <DatepickerControl
```

Comment:
Use `showUTC` prop here so the UTC issue gets fixed automatically when the base `DatePickerControl` component does.

Reply:
I can add it but I want to bring your attention to the fact the datepicker's label is `Release Time in UTC`  and adding the prop displays another `UTC` text above the datepicker, so it feels kind of redundant to have UTC twice. What do you think?